### ISSUE TITLE
Long method chaining in arguments will break indent

### ISSUE THREAD

**Issue by sseypt** [prettier/plugin-php#393]
Hello.

Which is in conflict with [PSR-2 method arguments](https://www.php-fig.org/psr/psr-2/#44-method-arguments)
> Argument lists MAY be split across multiple lines, where each subsequent line
> is indented once. When doing so, the first item in the list MUST be on the
> next line, and there MUST be only one argument per line.
> 
> When the argument list is split across multiple lines, the closing parenthesis
> and opening brace MUST be placed together on their own line with one space
> between them.

Input:
```php
<?php
someFunction($var, SomeClass::someFunction()->anotherFunction()->looooooooooongFunctionName());

```

Output:
```php
<?php
someFunction($var, SomeClass::someFunction()
    ->anotherFunction()
    ->looooooooooongFunctionName());

```

Expected:
```php
<?php
someFunction(
    $var,
    SomeClass::someFunction()
        ->anotherFunction()
        ->looooooooooongFunctionName()
);
```

Best regards,
Steve

**Comment by sseypt** [comment]
I'm not sure if this is related, but the same is true for ternary operator with comment.

Input:
```php
<?php
someFunction($a, $b ? $c /* comment comment comment comment comment comment comment comment */ : $d);
```

Output:
```php
<?php
someFunction($a, $b
    ? $c /* comment comment comment comment comment comment comment comment */
    : $d);
```

**Comment by sseypt** [comment]
I added the corresponding psr-2 paragraph.

**Comment by mgrip** [comment]
[JS formatting, for reference](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEBnCBbOAxArlMGAS2gAoASANwEMAnAGgAIBlTOAYQBtrVe28CxMgEoAdNSgQYACzi0BhElFJjOEdRs3QA5gqFQActSwrhAbgA6UK+ix6lFak3IAjRgH5G5MIwD0AKkZIDCxYIMxQmHCQhCjgyOiE+NjElOSw-19GJC8AE3MrKxB6EAgAB31UZFA6WggAdwAFOgQqlGpKCCJc4pAeGGQAM2pOVDgSl1pqMABrOBhmMumiKG1kGFpccZBciDAhkbGSlbHaGEap7QxqA9HtgCtUAA8AISnZ+eZjOAAZFbhbkdSrgYGUQQAmQHbJa0U7IEAuaguACeaigvTKtBWMAA6t0ZMgABwABhKmIgYxxUzK8MxcFOlABJVocAAjrgiCyLtQrjckMM7iUxhgiOtNttUCttJw4ABFXBSAH8w7bGBIvG5AlIcElDbUIicKXsCJ8kCSKBMkC4MYAFSRbQFYwAvk6gA) - seems to be the same as what @sseypt is suggesting

**Comment by mgrip** [comment]
should hopefully get fixed with https://github.com/prettier/plugin-php/pull/398

**Comment by mgrip** [comment]
Cormed fixed in #368 - @sseypt the output from your examples now match output from prettierjs, feel free to reopen if you think any more changes are needed and we can discuss
